### PR TITLE
feat(kernel): make messages translatable

### PR DESCRIPTION
**Pull request by obdulia-losantos**
:nigeria: :de: :gb: :es: :fr:

### CODE REVIEW

**Review by admbtlr — DISMISSED** [review]
🌍

**Comment by obdulia-losantos** [comment]
@adamvert could you review it again? I've just included more docs to the models :grimacing:

File: aether-couchdb-sync-module/aether/sync/couchdb/management/commands/setupcouchdb.py [hunk 1]
```diff
@@ -17,12 +17,14 @@
...
 
 from django.core.management.base import BaseCommand
+from django.utils.translation import ugettext as _
+
 from ...setup import setup_couchdb
 
 
 class Command(BaseCommand):
-    help = 'Setup couchdb'
+    help = _('Setup couchdb')
```
Comment (admbtlr) [inline aether-couchdb-sync-module/aether/sync/couchdb/management/commands/setupcouchdb.py]: Picking nits here, but... "setup" is a noun; the verb is "set up" with a space

**Review by admbtlr — APPROVED** [review]
👯‍♂️

**Pull request merged**